x-pack/plugins/monitoring/server/config.test.ts

```diff
@@ -97,6 +97,9 @@ describe('config schema', () => {
             "index": "metricbeat-*",
           },
           "min_interval_seconds": 10,
+          "package": Object {
+            "index": "metrics-*",
+          },
```

Comment:
What is the purpose of this setting ? The metricbeat one was added as a workaround for performance reasons according to [the documentation](https://www.elastic.co/guide/en/kibana/current/monitoring-settings-kb.html), do we need this in package context ?

Reply:
Thank you for pointing that out! I don't think is needed for the package - I removed it.